udata/tests/apiv2/test_datasets.py

```diff
@@ -53,6 +53,8 @@ def test_search_dataset(self):
         _dataset_org = DatasetFactory(organization=org)
         dataset_org_public_service = DatasetFactory(organization=org_public_service)
 
+        # This is needed or the test will fail if you have a search service configured in your udata.cfg file.
```

Comment:
This is unrelated to this PR, but this test will fail if you have a search service configured, which I needed to fix this PR ;)